clients/fides-js/src/docs/fides-event.ts

```diff
@@ -142,17 +142,35 @@ export interface FidesEvent extends CustomEvent {
       /**
        * What consent method (if any) caused this event.
        */
-      consentMethod?: "accept" | "reject" | "save" | "dismiss" | "gpc";
+      consentMethod?:
+        | "accept"
+        | "reject"
+        | "save"
+        | "dismiss"
+        | "acknowledge"
+        | "gpc"
+        | "script"
+        | "ot_migration";
```

Comment:
new methods have been added to the system, but hadn't been updated here. An unrelated docs update for clarity, consistency, and quality. Reminder note also added below to help catch this in the future.